resilience4j-timelimiter/src/main/java/io/github/resilience4j/timelimiter/TimeLimiter.java

```diff
@@ -54,26 +72,7 @@ static TimeLimiter of(Duration timeoutDuration) {
      * @return a future supplier which is restricted by a {@link TimeLimiter}.
      */
     static <T, F extends Future<T>> Callable<T> decorateFutureSupplier(TimeLimiter timeLimiter, Supplier<F> futureSupplier) {
-        return () -> {
```

Comment:
I moved the body to implementation class mainly because of `publishTimeLimiterEvent` call but also because all logic was in a static method in an interface instead of impl.